docker-compose.yml

```diff
@@ -8,6 +8,15 @@ services:
     volumes:
       - /var/lib/postgresql
     command: echo true
+    
+  nginx:
+    build: .
+    ports:
+        - "80:80"
```

Comment:
I don't understand the need of this. The docker-compose service "nginx" runs ./Dockerfile as the build command. 
Essentially this is the same as service "web". So why run 2 web services ? Also the ./Dockerfile runs nothing on port 80 so this should not do anything. 
**I think nginx has no role to play in our system.**
@shivamMg Can you please provide your views on this.